.github/workflows/vademecum.yaml

```diff
@@ -0,0 +1,30 @@
+name: Vademecum
+on: [push]
+
+jobs:
+  Runningscript:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+    
+      - name: Update repos
+        run: sudo apt update
+    
+      - name: Install poppler-utils
+        run: sudo apt install poppler-utils
+
+      - name: Install wkhtmltopdf
+        run: sudo apt install xvfb libfontconfig wkhtmltopdf
+
+      - name: Making script executable
+        run: chmod +x scripts/build.sh
+      
+      - name: Running deployment script
+        run: sh ./scripts/build.sh
+
+      - uses: mikeal/publish-to-github-action@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
Cambialo in `GITHUB_TOKEN` così utilizza quello automatico del repo invece di `TOKEN`

Reply:
secondo me non va, ma ci ho provato ugualmente